homeassistant/components/blebox/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "BleBox devices",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/blebox",
-  "requirements": ["blebox_uniapi==2.1.0"],
+  "requirements": ["blebox_uniapi==2.1.3"],
```

Comment:
The version in `setup.py` isnt' updated:
https://github.com/blebox/blebox_uniapi/blob/0259216782ac8e8f9f4f89d56dd0a8ee93fff841/setup.py#L26

Reply:
@MartinHjelmare fixed, thanks https://github.com/blebox/blebox_uniapi/blob/5da2373af2e9cbd52426729045b3a467e9d2ef0e/setup.py#L26

Reply:
Doesn't that need re-publish with a new release? The sources on pypi aren't updated?

Reply:
Release is the same(version). Before package built I changed mentioned line in blebox_uniapi. There is no difference between repository and PyPi. I've missed that one commit.

Reply:
Confirmed, thanks! 👍